pkg/cli/clivalidation.go

```diff
@@ -270,13 +270,21 @@ func RequireResourceType(args []string) (string, error) {
 	}
 	resourceTypeName := args[0]
 	supportedTypes := []string{}
+	foundTypes := []string{}
 	for _, resourceType := range clients.ResourceTypesList {
 		supportedType := strings.Split(resourceType, "/")[1]
 		supportedTypes = append(supportedTypes, supportedType)
-		if strings.EqualFold(supportedType, resourceTypeName) {
-			return resourceType, nil
+		//check to see if the resource type is the correct short or long name.
+		if strings.EqualFold(supportedType, resourceTypeName) || strings.EqualFold(resourceType, resourceTypeName) {
```

Comment:
Could you add a unit test for this change?

Reply:
@vinayada1,  it doesn't look like any of these functions in clivalidation are being independantly unit tested.. Do you want me to test this in the resources module (its used by list, show and delete) or should we start a clivalidation_test.go file and start it with this test?

Reply:
You are right! It would be great if you could start with a clivalidation_test.go file. Thanks!